technic/tools/init.lua

```diff
@@ -10,7 +12,7 @@ if enabled("flashlight")        then dofile(path.."/flashlight.lua") end
 if enabled("cans")              then dofile(path.."/cans.lua") end
 if enabled("chainsaw")          then dofile(path.."/chainsaw.lua") end
 if enabled("tree_tap")          then dofile(path.."/tree_tap.lua") end
-if enabled("sonic_screwdriver") then dofile(path.."/sonic_screwdriver.lua") end
+if enabled("sonic_screwdriver") and mesecons_materials and not has_mcl then dofile(path.."/sonic_screwdriver.lua") end
```

Comment:
MCL already has a screwdriver in the game

Reply:
So has MTG, point of sonic screwdriver isn't replacing regular screwdriver. It handles things completely different way and utilizes technic energy charge.

Reply:
ok

Reply:
@nonfreegithub should also remove mesecons_materials check (basically revert whole tools/init.lua to its original state). Only thing it uses from mesecons_materials is crafting recipe ingredients and those do not have to be valid. Missing ingredients just causes recipe to be ignored but tool can still be obtained.

If you think that screwdriver might cause real issues then, instead of adding hard checks, make it conditionally disabled by default based on installed mods but allow enabling it if server owner wants to do so.
If needed, this can be done here: https://github.com/mt-mods/technic/blob/f1b99282e889e9a483c76af95ee3370b6d2b2d04/technic/config.lua#L3-L14 at line 12

Reply:
Conditional default config probably isn't needed anyway as sonic_screwdriver anyway wont be available without `/give` privileges when mesecons_materials isn't installed.